src/GrpcProtos/capture.proto

```diff
@@ -467,15 +611,26 @@ message ClientCaptureEvent {
     // use them for high frequency events. For the rest please assign
     // numbers starting with 16.
     //
-    // Next high-frequency ID: 10
-    // Next lower-frequency ID: 38
+    // Next high-frequency ID: -- out of numbers
+    // Next lower-frequency ID: 43
     // Please keep these alphabetically ordered.
 
     // Even though AddressInfo is a high-frequency event
     // it is going to go away in the future when we switch to
     // frame-pointer based unwinding.
     AddressInfo address_info = 16;
     ApiEvent api_event = 9;
+    ApiScopeStart api_scope_start = 10;
+    ApiScopeStartAsync api_scope_start_async = 11;
+    ApiScopeStart api_scope_stop = 12;
+    ApiScopeStartAsync api_scope_stop_async = 13;
+    ApiStringEvent api_string_event = 14;
```

Comment:
maybe move bigger events beyond the threshold and keep smaller ones in? Or is string_event more likely to appear here?

Reply:
Hard to judge. The `api_string_event` is highly coupled to the async scopes, which from gut feeling will be used more frequently then the value tracking.

Reply:
In general manual instrumentation is mostly about synchronous scopes, I would only keep `ApiScopeStart` and `ApiScopeStop` here in the high frequency range.

Reply:
Fine for me.

Reply:
Done.